hackathon_site/dashboard/frontend/src/components/inventory/InventoryFilter/InventoryFilter.js

```diff
@@ -12,79 +12,74 @@ import FormControlLabel from "@material-ui/core/FormControlLabel";
-                                name="inventoryFilter"
-                                value={`category-${item.id}`}
-                                label={item.name}
-                                control={<Checkbox color="primary" />}
-                            />
-                            <Chip
-                                label={item.qty}
-                                className={styles.filterCategoryChip}
-                            />
-                        </div>
-                    ))}
-                </fieldset>
-            </form>
-        </Paper>
+                    <FormControlLabel
+                        name="orderBy"
+                        value="a-z"
+                        label="A-Z"
+                        control={<Radio color="primary" />}
+                    />
+                    <FormControlLabel
+                        name="orderBy"
+                        value="z-a"
+                        label="Z-A"
+                        control={<Radio color="primary" />}
+                    />
+                    <FormControlLabel
+                        name="orderBy"
+                        value="stock-high-low"
+                        label="Stock remaining: high to low"
+                        control={<Radio color="primary" />}
+                    />
+                    <FormControlLabel
+                        name="orderBy"
+                        value="stock-low-high"
+                        label="Stock remaining: low to high"
+                        control={<Radio color="primary" />}
+                    />
+                </RadioGroup>
+            </fieldset>
+            <Divider className={styles.filterDivider} />
+            <fieldset>
+                <legend>
+                    <Typography variant="h2">Availability</Typography>
+                </legend>
+                <FormControlLabel
+                    name="inStock"
+                    value="in-stock"
+                    label="In stock"
+                    control={<Checkbox color="primary" />}
+                />
+            </fieldset>
+            <Divider className={styles.filterDivider} />
+            <fieldset>
+                <legend>
+                    <Typography variant="h2">Categories</Typography>
+                </legend>
+                {categories.map((item, i) => (
+                    <div className={styles.filterCategory} key={i}>
+                        <FormControlLabel
+                            name="inventoryFilter"
+                            value={`category-${item.id}`}
+                            label={item.name}
+                            control={<Checkbox color="primary" />}
+                        />
+                        <Chip label={item.qty} className={styles.filterCategoryChip} />
+                    </div>
+                ))}
+            </fieldset>
+        </form>
         <div className={styles.filterBtns}>
             <Button
```

Comment:
Once again I can't comment on the actual line, but should `fullWidth={true}` be set on the reset button? It's noticeable when being clicked with the animation

Reply:
It was meant to be smol